rules/okta_rules/okta_user_mfa_reset_all.py

```diff
@@ -0,0 +1,17 @@
+from panther_base_helpers import okta_alert_context
+
+
+def rule(event):
+    return event.get("eventtype") == "user.mfa.factor.reset_all"
+
+
+def title(event):
+    return (
+        "Okta all authentication factors were reset for "
```

Comment:
```suggestion
        "Okta: All authentication factors were reset for "
```

Reply:
Similar feedback on this - lets put this in "english" - its not clear if its talking about MFA factors or what